mir_eval/segment.py

```diff
@@ -458,58 +492,40 @@ def rand_index(
         length (in seconds) of frames for clustering
         (Default value = 0.1)
     beta : float > 0
-        beta value for F-measure
-        (Default value = 1.0)
+        deprecated parameter - to be removed in 0.9 !!!
```

Comment:
Why is this deprecated?

Reply:
Hrm, indeed.  I wonder how this ended up there in the first place?

I agree that it makes sense to deprecate this.  However, I think we should take that to a separate PR, and add in a bit of machinery to support deprecations better.  (See, e.g., https://github.com/librosa/librosa/blob/ebd878fabb87c588f98f372a94fca5d167c59cf2/librosa/util/deprecation.py#L10-L15 )